frontend/app/views/spree/shared/_footer.html.erb

```diff
@@ -53,16 +53,16 @@
           <% spree_menu('footer').children.each do |parent| %>
             <div class="col-3">
               <div class="footer-spree-label">
-                <%= link_to parent.name, spree_localized_item_link(parent) %>
+                <% if parent.container? %>
+                  <%= parent.name %>
+                <% else %>
+                  <%= spree_nav_link_tag(parent) %>
+                <% end %>
               </div>
               <% parent.children.each do |child| %>
-                <% if child.code == 'category' && child.item_type == 'Container' %>
-                  <% if child.children.present? %>
-                    <% child.children.each do |category_link| %>
```

Comment:
@MatthewKennedy shouldn't we have this iteration?

Reply:
I think the footer menu should be simple as possible, just a set of links parent links, with links nested, no need for containers here.

This could be a shining example of how simple a menu can be, vs the more complex main menu that uses containers for promos and categories.